paper.tex

```diff
@@ -139,19 +142,21 @@ \section*{Ancestral graphs and stochastic processes}
 and the model has proved to be an especially powerful basis for
 simulations~\citep{hudson2002generating,baumdicker2021efficient}.
 \cite{kelleher2016efficient} provide a description of his algorithm along with
-an efficient implementation, the details of which turn out to be highly relevant in 
+an efficient implementation, the details of which turn out to be highly relevant to 
 later sections of the current paper (see the ``\nameref{Ancestry_resolution}'' section).
 
-The Ancestral Recombination Graph (ARG) was later introduced by
-Griffiths~\citep{griffiths1991two,ethier1990two,griffiths1996ancestral,
-griffiths1997ancestral}, and is a
-closely related and complementary treatment of the same process. Where
-Hudson's algorithm is focused on the efficient \emph{simulation} of the
-process, the original ARG literature is focused on mathematical
-results about the stochastic process. Griffiths and colleagues used the word
+Although Hudson's algorithm was accompanied by a graph
+schematic~\citep[][Fig. 5]{hudson1983properties}, the term
+"Ancestral Recombination Graph" or ARG, was introduced later by
```

Comment:
```suggestion
``Ancestral Recombination Graph'' or ARG, was introduced later by
```

Reply:
Damn, I thought I had caught all those! Thanks for spotting.

Reply:
The GitHub diff shows them up very clearly